research-plan.md

```diff
@@ -14,23 +14,24 @@ The goal of this research is to inform hypotheses that address the following:
 
 ## User groups
 
-Prioritization is a function of how much insight I believe we have into a user group.
-
 ### Requesters
 
-- Business/Industry (High priority)
-- Attorneys (High priority)
-- Individual one-off requesters (High priority)
-- Civic society (Low priority)
-- Journalists (Low priority)
+- Business/Industry and regulated community
```

Comment:
Even though we're basing prioritization on where our research is currently thin, I'm worried this will come off as overall prioritization and be a bit misleading.